minvws/nl-kat-coordination
# Fix async code calling blocking sync code

### Changes

This fixes async functions calling blocking sync code. After spotting this in a code review, I checked and saw that we already did this incorrectly in a few other places. Async code should not call blocking sync code because that will block the whole async event loop.

### QA notes

It is hard to verify that it was not correct before, because the only effect is that the code would block in a place where it shouldn't.

---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
